src/databricks/labs/ucx/azure/resources.py

```diff
@@ -94,6 +115,18 @@ class AzureRoleAssignment:
     role_name: str
 
 
+@dataclass
+class AccessConnector:
+    id: str
+    name: str
+    location: str
```

Comment:
```suggestion
    location: str
    resource_group: str
    subscription_id: str
```
just in case.

Reply:
Can be easily retrieved using the id field as changed in the other comment to AzureResource